src/module.c

```diff
@@ -12210,6 +12210,76 @@ int RM_LoadConfigs(RedisModuleCtx *ctx) {
     return REDISMODULE_OK;
 }
 
+int RM_RdbLoad(const char *filename, int flags) {
+    errno = 0;
+
+    if (flags != 0) {
+        errno = EINVAL;
+        return REDISMODULE_ERR;
+    }
+
+    /* TODO:
+     * - Call fsync for filename
+     * - Stop AOF?
+     * - Kill RDB fork?
+     */
```

Comment:
yes, let's stop the aof (restart later), and let's kill the rdb fork (storing outdated data, and creating COW mess).

why would we want to fsync before storing anything?

Reply:
done